.eslintrc.cjs

```diff
@@ -34,6 +35,49 @@ const config = {
     },
   },
   rules: {
+    '@cspell/spellchecker': [
+      'error',
+      {
+        cspell: {
+          words: [
```

Comment:
instead of keeping this list that long, could we not just replace some of these that are only used in tests or local variables with known words?

Reply:
I'll try it soon.

Reply:
I made it simpler than before in [c8770ba](https://github.com/TanStack/query/pull/6718/commits/c8770ba6d3e72f02eadf92be89faf45c7cfe193a)